webapp/src/main/webapp/templates/freemarker/body/fileUpload/fileUpload-default.ftl

```diff
@@ -0,0 +1,32 @@
+<#-- $This file is distributed under the terms of the license in LICENSE$ -->
+
+
+${scripts.add('<script type="text/javascript" src="${urls.base}/js/fileUpload/fileUploadUtils.js"></script>')}
+
+<#assign i18n = i18n() >
+
+<section id="fileploadContainer" role="region">
+    <h2>${i18n.file_upload_heading}</h2>
+
+    <#if errorMessage??>
+        <section id="error-alert" role="alert"><img src="${urls.images}/iconAlert.png" alt="${i18n.alt_error_alert}" />
+            <p>${errorMessage}</p>
+        </section>
+    </#if>
+	<#if action?? && action == "upload" >
+	    <form id="fileUploadForm" action="${formAction}" enctype="multipart/form-data" method="post" role="form">
+	        <label>${i18n.upload_files_supported_types} 
+	        <#if supportedTypes??>
+	        	<span>${supportedTypes}</span></label>
+			</#if>
+			<#if supportedMIMETypes??>
+	        	<span>MIME Types: ${supportedMIMETypes}</span></label>
```

Comment:
MIME Types not localized (i18n). Do we need both supportedTypes and supportedMIMETypes?

Reply:
supportedTypes is a list of allowed file extensions for user's convenience.
supportedMIMETypes are less easier to read ( image/png, application/pdf )but more strict definition of file types allowed by configuration.
I decided to provide both for regular and experienced users.
I don't think we should localize mime types.

Reply:
I believe @chenejac is suggesting the text `MIME Types:` itself should be i18n, not the type list of options. Looking at wiki (e.g. [tipos MIME](https://es.wikipedia.org/wiki/Tipo_de_medio) in spanish), I think it would be beneficial to have it internationalized.

Reply:
Thanks, Benjamin. Sure, I will fix that and prepare PRs for VIVO-languages and Vitro-languages.

Reply:
I improved internationalization, naming, removed extensions information in favor of media types.